decidim-admin/app/commands/decidim/admin/bulk_block_users.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Admin
+    class BulkBlockUsers < Decidim::Command
+      # Public: Initializes the command.
+
```

Comment:
```suggestion

      # Public: Initializes the command.
```

Reply:
Rubocop is not letting me add an indentation here.